app/assets/stylesheets/components/_tree.scss

```diff
@@ -1,3 +1,25 @@
-.cor {
-  color: yellow;
+.container-header {
+  display: flex;
+  justify-content: center;
+  height: 100px;
+  width: 100%;
+  background-color: darkgreen;
+  color: gold;
+  border-bottom: solid 2px gold;
+}
+
+.header {
```

Comment:
Why arent we using BEM?

Reply:
Because today's purpose is to learn git command and not to learn BEM or sass, even with we are using .scss files.